### PR TITLE
[Arm-193] Open source Contributing guide & Code of conduct

### PR DESCRIPTION
**Pull request by JasterV**
Stills in progress

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Contributing to Armore
+
+🍻🎉 Welcome! First off, thanks for taking the time to contribute! 🎉🍻
+
+Feel free to contribute regardless of your age, gender, race or any other discriminating factor (Check out [code of conduct](./CODE_OF_CONDUCT.md) for more) and contact us on [discord](https://discord.gg/mptG9ZGxTF) if you have any question, we are glad to have you here!
```
Comment (griffobeid) [inline CONTRIBUTING.md]:
```suggestion
Feel free to contribute regardless of your age, gender, race or any other discriminating factor (Check out [code of conduct](./CODE_OF_CONDUCT.md) for more) and contact us on [discord](https://discord.gg/mptG9ZGxTF) if you have any questions, we are glad to have you here!
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## How to contribute?
+
+Armore is divided into 4 main repositories: _Android_, _iOs_, _Server_ & _Web_.
```
Comment (griffobeid) [inline CONTRIBUTING.md]:
```suggestion
Armore is divided into 4 main repositories: _Android_, _iOS_, _Server_ & _Web_.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+For now we are mainly open sourcing the Server repo to the community. If you are interested in collaborating with the android, iOs or Web code (which we will open soon) contact us!
```
Comment (griffobeid) [inline CONTRIBUTING.md]:
```suggestion
For now we are mainly open sourcing the Server repo to the community. If you are interested in collaborating with the android, iOS or Web code (which we will open soon) contact us!
```

**Review by griffobeid — CHANGES_REQUESTED** [review]
Just a few minor things, this looks awesome otherwise 👏 💯

**Review by darioalessandro — APPROVED** [review]
lgtm!

**Review by griffobeid — APPROVED** [review]

**Pull request merged**